documentation/src/main/markdown/packagemgrs/cpan.md

```diff
@@ -0,0 +1,18 @@
+# CPAN Support
+
+## Related properties
+
+[Detector properties](../properties/detectors/cpan.md)
+
+## Overview
+
+The CPAN detectable will run if it finds a Makefile.PL file.
+
+The detectable requires the following executables:
+
+* cpan - used to determine the list of direct dependencies required by the project
+* cpanm - used to determine the list of installed Perl modules on the system in order to assign versions to the dependencies found by cpan
```

Comment:
Tweak suggestion: "used to assign versions to the dependencies found by cpan by determining the list of Perl modules installed on the system."